packages/components/input/src/input.vue

```diff
@@ -368,7 +369,7 @@ const createOnceInitResize = (resizeTextarea: () => void) => {
     const isElHidden = textarea.value?.offsetParent === null
     if (!isElHidden) {
       resizeTextarea()
-      isInit = true
+      rAF(() => (isInit = true))
```

Comment:
After the change the closure will be triggered twice, i investigate a bit and it looks good but i'm feel like we can improve the solution.

```suggestion
     setTimeout(resizeTextarea)
      isInit = true
```
This can solve the solution but i don't think it's the right place to do this.
Maybe here:
https://github.com/cszhjh/element-plus/blob/4555f1bf6fb890e3ab3d408ec3fad6478419efda/packages/components/input/src/input.vue#L505

Reply:
I submitted a bit hastily, but it’s okay. The Playground is still at this commit: [1a8c7bb](https://github.com/element-plus/element-plus/pull/21840/commits/1a8c7bb364fa407adb97ca7dcdc3e09e7ce38f40).

The CI failure actually gives a perfect opportunity to test the scenarios I mentioned above 😄

Reply:
I think the callback [here](https://github.com/element-plus/element-plus/blob/2e19e59e67cc55f6b2febff156330f86d9f75415/packages/components/splitter/src/hooks/useSize.ts#L37) is executed once before and once after `onMounted`. 

The first time, `containerSize = 0`. Then `onMounted` and a `resize` event are triggered, followed by a second `resize` when `containerSize` changes.
 
I suspect the `setTimeout` inside `onMounted` might have already executed earlier. However, it’s hard for me to reproduce this locally 🤔

Reply:
I managed to reproduce this issue once during a local debugging session by chance, but I haven’t been able to find the reproduction path again.

<img width="760" height="374" alt="image" src="https://github.com/user-attachments/assets/3f4b0c47-5148-4415-9ffb-fcdb3dfb25a1" />

Reply:
Thanks for the explanation :bow: .